### PR TITLE
Add command in BUILD.md for Apple Silicon users to switch to a Rosetta shell

### PR DESCRIPTION
**Pull request by emiyl**
Should fix any confusion, such as in #334 and #333.

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -67,6 +67,14 @@ LLVM 15 isn't supported due to compatibility issues with Boost dependency. The O
 API isn't support on MacOS, Vulkan must be used. Additionally Vulkan must be used through the 
 Molten-VK compatibility layer.  
 
+### Apple Silicon
+
+Cemu is an x86 (Intel) application and must be run through Rosetta. As such, you must also compile it under Rosetta. If you are running an ARM (Apple Silicon) based Mac, run the following command in your terminal before continuing.
+
+`arch -x86_64 zsh`
```
Comment (UltraHDR) [inline BUILD.md:74]: `eval "$(/usr/local/Homebrew/bin/brew shellenv)"` needs to be run afterwards

**Comment by UltraHDR** [comment]
I've created an existing pull request here https://github.com/cemu-project/Cemu/pull/280/files

File: BUILD.md (hunk 1 above)
Comment (emiyl) [reply BUILD.md:74]: For simplicity, should I just add this to the "Installing brew" section? I don't think it'll cause any issues with Intel Macs. It's a little redundant though I suppose.

Comment (GumbyXGames) [reply BUILD.md:74]: I agree that `arch -x86_64 zsh` should just be added to the build steps.

Comment (emiyl) [reply BUILD.md:74]: Sorry if there was confusion, in my comment I meant the `eval "$(/usr/local/Homebrew/bin/brew shellenv)"` command should be added to "Installing brew", not the arch command.

**Comment by GumbyXGames** [comment]
I wrote two bash scripts for to perform a "fresh" build (install x86_64 brew and dependencies to the correct paths, clone repo, and run cmake commands) and a update and build script:

Fresh_build.sh: [](https://pastebin.com/adfZsjsp)
Update_and_Build.sh: [](https://pastebin.com/Rh894mD)e

**Pull request closed without merging**